src/Controller/TeamController.php

```diff
@@ -461,4 +462,37 @@ public function teamDelete(
             ]);
         }
     }
+
+    #[Route(path: '/team_custom/ignore', name: 'preset_set_ignored')]
+    public function setPresetIgnored(
+        Request                $request,
+        UrlGeneratorInterface  $urlGenerator,
+        TeamRepository         $teamRepository,
+        EntityManagerInterface $em,
+        InheritanceService     $inheritanceService
+    ): RedirectResponse
+    {
+        $team = $request->get('team');
+        $preset = $request->get('preset');
+        $type = $request->get('type');
+        $ignored = $request->get('set');
+
+        if (is_numeric($team)) {
+            $team = $teamRepository->find($team);
+        }
+
+        if (is_numeric($preset)) {
+            $preset = $em->getRepository($type)->find($preset);
+        }
+
+        if ($team && $preset) {
+            $inheritanceService->setIgnored($preset, $team, $ignored);
+            $em->persist($preset);
+            $em->persist($team);
+            $em->flush();
+        }
+
+        $referer = $request->headers->get('referer');
+        return new RedirectResponse($referer ?: $urlGenerator->generate('dashboard'));
+    }
```

Comment:
I think we need here some access control. In the moment I can send requests to that controller, as long as I am logged in. But only team admins should be able to use this endpoint.

And it should actually be a PUT request and not a GET request, as it is a good practice to not change data at GET requests.